jerry-main/main-unix.c

```diff
@@ -150,8 +219,101 @@ print_unhandled_exception (jerry_value_t error_value)
   {
     jerry_size_t sz = jerry_string_to_char_buffer (err_str_val, err_str_buf, err_str_size);
     assert (sz == err_str_size);
+    err_str_buf[err_str_size] = 0;
+
+    if (jerry_value_is_syntax_error (error_value))
+    {
+      uint32_t err_line = 0;
+      uint32_t err_col = 0;
+
+      /* 1. parse column and line information */
+      for (uint32_t i = 0; i < sz; i++)
+      {
+        if (!strncmp ((char *) (err_str_buf + i), "[line: ", 7))
+        {
+          i += 7;
+
+          char num_str[8];
+          uint32_t j = 0;
+
+          while (i < sz && err_str_buf[i] != ',')
+          {
+            num_str[j] = (char) err_str_buf[i];
+            j++;
+            i++;
+          }
+          num_str[j] = '\0';
+
+          err_line = str_to_uint (num_str);
+
+          if (strncmp ((char *) (err_str_buf + i), ", column: ", 10))
+          {
+            break; /* wrong position info format */
```

Comment:
IMHO, break is better here, because on a return there is no error message printing, so if you don't check the exit value, then you don't even notice the program failure. Check the end of the function.